### PR TITLE
New review system 

### PR DESCRIPTION
**Pull request by jthistle**
https://trello.com/b/OoU9Px0U/

There aren't any multiple reviews or reviews with star ratings, but here is what they look like:

Screenshot from 2021-03-17 12-13-40

then, after pressing 'next':

Screenshot from 2021-03-17 12-13-54

### CODE REVIEW

**Comment by marxian** [comment]
#### 📝Changed routes:


| [**/service/\[id\]**](https://my-best-life-en44orv0j-neontribe.vercel.app/service/[id]) |
|:-:|
|<a href="https://my-best-life-en44orv0j-neontribe.vercel.app/service/[id]">Screenshot of /service/[id]</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/_WtEQ5A__w8FiJE0Bjo_aSsvdMk_.png">(view full size)</a> |







Commit 2b22aa687c38940046664b8ed66cc200a40b3a29 (https://my-best-life-en44orv0j-neontribe.vercel.app).

File: .prettierignore [hunk 1]
```diff
@@ -2,4 +2,5 @@ node_modules
 .next
 yarn.lock
 package-lock.json
-public
\ No newline at end of file
+public
+content
```
Comment (Nikomus) [inline .prettierignore:6]: 👍

**Review by edpe — APPROVED** [review]
looks good :ship:

File: src/Components/Carousel.tsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  width: 100%;
+  display: flex;
+  justify-content: space-evenly;
+  margin-top: 1rem;
+`
+
+const Movement = styled.button`
+  background: none;
+  border: none;
+  color: black;
```
Comment (Nikomus) [inline src/Components/Carousel.tsx:23]: Extra color here - I tend to alphabetise the rules and then move order up/down for overrides in longer CSS blocks

File: src/Components/Carousel.tsx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+  }
+
+  return (
+    <CarouselContainer>
+      {children &&
+        React.Children.map(children, (child, i) => {
+          if (i === currentIdx) {
+            return <Show>{child}</Show>
+          }
+          return <Hide aria-hidden={true}>{child}</Hide>
```
Comment (Nikomus) [inline src/Components/Carousel.tsx:72]: If these are hidden and stacked on top of each other, will the height be defined by the largest review?

**Review by Nikomus — COMMENTED** [review]
We can try this for now, but there's quite a bit here that is not planned. Carousels don't have great usability and they're only appropriate for particular volumes of content - which is currently unknown. We should check in on the board if we think there are new elements that we need to add to make a ticket work.

**Comment by jthistle** [comment]
Yeah, the carousel was just my natural response to being asked to implement a way of displaying an arbitrarily long list of reviews - I'm 100% open to the idea of replacing it with something else. I don't think the way it is currently is too bad usability-wise, personally, but maybe there is a better way of doing it.

Let's leave this unmerged for now until we have some input from the board :)

**Comment by jthistle** [comment]
@Nikomus replaced carousel with list, can I merge this now?

**Review by Nikomus — APPROVED** [review]
@jthistle Yep, let's go for this 👍

**Pull request merged**